internal/cmd/tui.go

```diff
@@ -133,7 +133,14 @@ func tuiCmd() *cobra.Command {
 						return err
 					}
 
-					err = runnerClient.RunBlock(ctx, result.block)
+					tempBlock := result.block.Block.Clone()
```

Comment:
```suggestion
					tempBlock := &result.block.Block.Clone()
```

This is only ever accessed as a pointer, so you might as well address here

Reply:
![image](https://github.com/stateful/runme/assets/2709271/f39d7d0c-82f1-483d-9338-a154c7b561c8)
It is now part of the clone method. So I removed the pointers here.